docs/src/docs/contributing/faq.mdx

```diff
@@ -12,7 +12,10 @@ See [there](/contributing/new-linters#how-to-add-a-private-linter-to-golangci-li
 
 ## How to update an existing linter
 
-Just update its version in `go.mod`.
+We use Dependabot to update dependencies (linters included).
+The updates happen at least automatically once a week (Sunday 11am UTC).
+
+No pull requests to update a linter will be accepted unless you are the author of the linter AND specific changes are required.
```

Comment:
In Markdown, we can emphasize by using `**and**`, but I'm against it. We should be polite to everyone reading the document.
```suggestion
No pull requests to update a linter will be accepted unless you are the author of the linter and specific changes are required.
```